packages/functions/src/controllers/asset-price.ts

```diff
@@ -0,0 +1,131 @@
+import { ChainSupportedAssets, SupportedChains } from '@midas-capital/types';
+import { BigNumber, constants, utils } from 'ethers';
+import { functionsAlert } from '../alert';
+import { environment, supabase } from '../config';
+import { MidasSdk } from '@midas-capital/sdk';
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { Handler } from '@netlify/functions';
+import {
+  arbitrum,
+  basegoerli,
+  bsc,
+  chainIdToConfig,
+  chapel,
+  ethereum,
+  evmos,
+  fantom,
+  ganache,
+  moonbeam,
+  neondevnet,
+  polygon,
+} from '@midas-capital/chains';
+import axios from 'axios';
+
+export const chainSupportedAssets: ChainSupportedAssets = {
```

Comment:
Instead of always defining this, please use: https://github.com/midas-protocol/monorepo/blob/development/packages/chains/src/index.ts#L17